rust/connlib/clients/android/src/lib.rs

```diff
@@ -22,6 +22,11 @@ use thiserror::Error;
 use tracing_subscriber::prelude::*;
 use tracing_subscriber::EnvFilter;
 
+/// The Android client doesn't use platform APIs to detect network connectivity changes,
+/// so we rely on connlib to do so. We have valid use cases for headless Android clients
+/// (IoT devices, point-of-sale devices, etc), so try to reconnect for 7 days.
+const MAX_PARTITION_TIME_SECS: Duration = Duration::from_secs(60 * 60 * 24 * 7);
```

Comment:
These could all just be `MAX_PARTITION_TIME` since they're a Duration, not an integer number of seconds, right?

Reply:
Good catch!